scripts/less-watch

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+/**
+
+Usage:
+./scripts/less-watch [watchPath]
+
+Example:
+./scripts/less-watch ./notebook/static/notebook/less
+
+**/
+
+var less = require('less');
+var fs = require('fs');
+var path = require('path');
+var child_process = require('child_process');
+
+function watchDir(dir) {
+  var rootPath = path.join(__dirname, '..');
+  var watchPath = path.resolve(dir);
+  console.log('less-watch:', 'watching:', path.relative(rootPath, watchPath));
+  fs.watch(watchPath, {recursive: true}, function(event, file) {
+    if (file && /.+\.less$/.test(file)) {
+      console.log('less-watch:', 'modified:', file);
+      child_process.exec('lessc --include-path=notebook/static --verbose notebook/static/style/style.less notebook/static/style/style.min.css', function(err, stdout, stderr) {
```

Comment:
1. I tried that and running any of the NPM scripts from this script failed. On second thought, this is probably because it's trying to run it from the `/scripts/` directory vs. the root. I'll give this a quick try...
2. lessc is an devDependecy, so all devs should have it installed locally within the project. NPM will add `./node_modules/.bin` to the user's PATH when running an NPM script. Case in point, I don't have lessc installed globally and this works.